model/src/main/java/com/basistech/rosette/dm/AttributeKey.java

```diff
@@ -52,8 +52,8 @@
     TRANSLATED_TOKENS("translatedTokens"),
     TOPIC_RESULTS("topicResults"),
 
-    CONCEPT("concepts"),
-    KEYPHRASE("keyphrases");
+    CONCEPT("concept"),
```

Comment:
Happy to default to Hannah's judgement here, but I disagree with this change.  This is the top level key, meaning jq users will access the (usually list of) results with `jq .attributes.<attribute_key>`.  IMO this should be the plural form since the return value (in this case) is a list of concepts/keyphrases and not a single entry.